File: src/script_resources/result.cpp
#include <iostream>
#include <fstream>
#include <string>
#include "text_conversion.h"

int main()
{
   //  this is some long script
   std::string text = "this is some test headline to process bla bla";

   text_conversion::convert_to_title_case(text);

   std::cout << text << std::endl;

   {
      const std::string temp((std::istreambuf_iterator<char>(std::ifstream("my_data.txt").rdbuf())), std::istreambuf_iterator<char>());
      text = temp;
   }

   std::cout << text << std::endl;

   text_conversion::convert_to_title_case(text);

   std::cout << text << std::endl;

   std::ofstream("my_data.txt") << text;

   return 0;
}

File: src/script_resources/result.py
import text_conversion

if __name__ == '__main__':
   # this is some long script
   text = "this is some test headline to process bla bla"
   text = text_conversion.title_case(text)
   print(text)
   with open("my_data.txt", 'r') as file:
      text = file.read()
   print(text)
   text = text_conversion.title_case(text)
   print(text)
   with open("my_data.txt", "w") as file:
      file.write(text)

